internal/providers/terraform/azure/testdata/container_registry_test/container_registry_test.golden

```diff
@@ -0,0 +1,35 @@
+
+ Name                                           Monthly Qty  Unit            Monthly Cost 
+                                                                                          
+ azurerm_container_registry.basic                                                         
+ ├─ Registry usage (Basic)                               30  days                   $5.00 
+ ├─ Storage (over 10 GB)                                140  GB-months             $14.00 
+ └─ Build vCPU                                      540,000  seconds               $54.00 
+                                                                                          
+ azurerm_container_registry.my_registry                                                   
+ ├─ Geo replication (1 location)                         30  days                  $50.00 
+ ├─ Registry usage (Premium)                             30  days                  $50.00 
+ └─ Build vCPU                                      540,000  seconds               $54.00 
+                                                                                          
+ azurerm_container_registry.overStorage                                                   
+ ├─ Registry usage (Basic)                               30  days                   $5.00 
+ ├─ Storage (over 10 GB)                                540  GB-months             $54.00 
+ └─ Build vCPU                                      540,000  seconds               $54.00 
+                                                                                          
+ azurerm_container_registry.twoLocations                                                  
```

Comment:
"If the usage file value is above the "Included storage" then apply the Additional storage cost, otherwise this should show as a usage-based cost component."

So Storage should always be shown, if there is a usage file property then it should show costs, otherwise it should show "usage depend on cost..."

Reply:
@alikhajeh1 Storage should always be shown, if there is a usage file property and value above "Included strorage", right?

Reply:
hmm good point, so there are 3 cases, here's what I think is nice:
1. no usage file: show "Storage (over XGB)" and "cost depends on usage"
2. usage file with value less than included storage. Don't show Storage cost as it's free.
3. usage file with value more than included storage. Show "Storage (over XGB)" and costs.